cmd/da-light-client/start/start.go

```diff
@@ -156,7 +156,7 @@ func Cmd() *cobra.Command {
 
 func addFlags(cmd *cobra.Command) {
 	cmd.Flags().
-		StringP(rpcEndpointFlag, "", "mocha-4-consensus.mesa.newmetric.xyz", "The DA rpc endpoint to connect to.")
+		StringP(rpcEndpointFlag, "", "http://localhost:8000", "The DA rpc endpoint to connect to.") // TODO : change the DA rpc if required
```

Comment:
please remove occurances where avail is set as the default da to replace celestia, currently it's not tested/integrated into roller and isn't considered stable